# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: tools/lint/rustfmt.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/tools/lint/rustfmt.yml b/tools/lint/rustfmt.yml
--- a/tools/lint/rustfmt.yml
+++ b/tools/lint/rustfmt.yml
@@ -3,17 +3,16 @@ rust:
     description: Reformat rust
     min_rustfmt_version: 1.4.12
     include:
         - '.'
     exclude:
         - dom/webauthn/libudev-sys/
         - gfx/wr/peek-poke/
         - gfx/wr/webrender_build/
-        - gfx/wgpu/
         - gfx/wr/wr_malloc_size_of/
         - media/mp4parse-rust/
         - servo/
     extensions:
         - rs
     support-files:
         - 'tools/lint/rust/**'
     type: external

